src/main/java/gov/nj/innovation/customAwsIdp/lambda/Handler.java

```diff
@@ -0,0 +1,76 @@
+package gov.nj.innovation.customAwsIdp.lambda;
+
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import gov.nj.innovation.customAwsIdp.keys.KeyConstants;
+import gov.nj.innovation.customAwsIdp.keys.KeysWrapper;
+import gov.nj.innovation.customAwsIdp.SamlGenerator;
+import org.apache.log4j.Logger;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+
+/**
+ * Handler modified from code generated by Serverless ({@code com.serverless.Handler}).
+ *
+ * @author Case Walker (case@innovation.nj.gov)
+ */
+public class Handler implements RequestHandler<Map<String, Object>, String> {
+
+    private static final Logger logger = Logger.getLogger(Handler.class);
+    private static final String DEFAULT_DURATION = System.getenv("DEFAULT_SESSION_DURATION");
+    private static final Pattern DIGITS_PATTERN = Pattern.compile("\\d+");
+
+    @Override
+    public String handleRequest(Map<String, Object> input, Context context) {
+        final String user = (String) input.get("user");
+        final String ssoRole = (String) input.get("ssoRole");
```

Comment:
@namanaman BTW I changed the argument name so it makes more sense, thanks for pointing out that the name wasn't clear before.